app/views/devise/confirmations/new.html.erb

```diff
@@ -0,0 +1,16 @@
+<h2>Resend confirmation instructions</h2>
```

Comment:
* As the previous CR pointed out, i get errors when i try to start the server below is a screenshot of the error i get. Kindly fix it 
![image](https://user-images.githubusercontent.com/52497246/182642920-4c012898-2688-43ce-bce6-b4392bb63b64.png)